README.md

```diff
@@ -35,6 +35,19 @@ To build, you will need the following in your PATH:
   - We recommend installing this via the
     [Visual Studio Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022)
 
+## Documentation
+
+Please refer to our [Contributing Docs](https://contributing.bitwarden.com/) for
+[getting started](https://contributing.bitwarden.com/getting-started/sdk/) instructions and
+[architectural documentation](https://contributing.bitwarden.com/architecture/sdk/).
+
+You can also browse the latest published documentation:
+
+- [docs.rs](https://docs.rs/bitwarden/0.5.0/bitwarden/) for the public SDK.
```

Comment:
We should probably point to `https://docs.rs/bitwarden/latest/bitwarden/` or `https://docs.rs/bitwarden/`, otherwise the link will be outdated once the next release comes out